packages/communication-react/review/stable/communication-react.api.md

```diff
@@ -849,6 +853,10 @@ export interface CallWithChatAdapterSubscriptions {
     // (undocumented)
     on(event: 'messageReceived', listener: MessageReceivedListener): void;
     // (undocumented)
+    on(event: 'messageEdited', listener: MessageEditedListener): void;
```

Comment:
It looks like for the new functionality only `messageDeleted` and `messsageReceived` events are used. Is `messageEdited` event needed?

Reply:
It's not used by the chat badge button, but for developer observability it seems like an event we've just missed in exposing -- can remove if we don't want to expose it though

Reply:
`It's not used by the chat badge button, but for developer observability it seems like an event we've just missed in exposing`
True! I'm okay to leave it but more feedback from the team may be needed